x/fswap/keeper/keeper.go

```diff
@@ -0,0 +1,44 @@
+package keeper
+
+import (
+	"fmt"
+
+	"github.com/tendermint/tendermint/libs/log"
+
+	"github.com/Finschia/finschia-sdk/codec"
+	sdk "github.com/Finschia/finschia-sdk/types"
+	"github.com/Finschia/finschia-sdk/x/fswap/types"
+	paramtypes "github.com/Finschia/finschia-sdk/x/params/types"
+)
+
+type (
+	Keeper struct {
+		cdc        codec.BinaryCodec
+		storeKey   sdk.StoreKey
+		memKey     sdk.StoreKey
+		paramstore paramtypes.Subspace
+	}
+)
+
+func NewKeeper(
+	cdc codec.BinaryCodec,
+	storeKey,
+	memKey sdk.StoreKey,
+	ps paramtypes.Subspace,
+) *Keeper {
+	// set KeyTable if it has not already been set
+	if !ps.HasKeyTable() {
+		ps = ps.WithKeyTable(types.ParamKeyTable())
+	}
+
+	return &Keeper{
+		cdc:        cdc,
+		storeKey:   storeKey,
+		memKey:     memKey,
+		paramstore: ps,
```

Comment:
why don't we use collection instead of legacy param store?

Reply:
Could you give me an example about collection? I don't know the collection...

Reply:
Please refer [here](https://github.com/cosmos/cosmos-sdk/tree/main/collections)

Reply:
I guess, `finschia-sdk` don't have `collections` yet.
How about using current style(old one) and update it when we start to work on `bump50`?